docs/translations/api-docs/date-pickers/date-calendar/date-calendar.json

```
@@ -15,7 +15,9 @@
     "defaultValue": {
       "description": "The default selected value. Used when the component is not controlled."
     },
-    "disabled": { "description": "If <code>true</code>, the picker and text field are disabled." },
+    "disabled": {
+      "description": "If <code>true</code>, the component will be disabled. When disabled, the value cannot be changed and no interaction is possible."
```

Nitpick: can we maintain present tense throughout all of this text? This is the simplest form.

```suggestion
      "description": "If <code>true</code>, the component is disabled. When disabled, the value cannot be changed and no interaction is possible."
```